drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig

```diff
@@ -7,89 +7,98 @@
  */
 #}
 <div class="avoindata-explore pb-4">
-  <div class="avoindata-explore-container">
-    <div class="row avoindata-explore-header-row"> 
+  <div class="avoindata-explore-container justify-content-center">
+    <div class="row avoindata-explore-header-row pt-3"> 
       <h2>
         {% trans %}
           Explore open data and its use cases
         {% endtrans %}
       </h2>
     </div>
-    <div class="row avoindata-explore-items d-flex flex-wrap justify-content-center pt-2 pb-2">
 
-      <div class="row avoindata-explore-card" container-fluid d-flex>
-        <div class="col-md-4 align-items-center d-flex avoindata-explore-card-icon dataset">
+    <div class="avoindata-explore-items mt-3 mb-3">
+
+      <div class="avoindata-explore-card mt-2">
+        <div class="avoindata-explore-card-icon avoindata-explore-dataset">
           <img src="/themes/avoindata/images/database.svg"/>
         </div>
 
-        <div class="col-md-8 avoindata-explore-card-content pl-3">
-          <div class="row container-fluid">
+        <div class="avoindata-explore-card-content">
+          <div class="avoindata-explore-card-title">
             <h3>
               {% trans %}
                 Datasets
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
+
+          <div class="avoindata-explore-card-text">
             <i class="far fa-chevron-right"></i> 
             <a href="/data/{{ language }}/dataset">
               {% trans %}
                Browse datasets
               {% endtrans %}
-               </a>
+            </a>
           </div>
         </div>
+
       </div>
 
-      <div class="row avoindata-explore-card" container-fluid d-flex>
-        <div class="col-md-4 align-items-center d-flex avoindata-explore-card-icon apiset">
+      <div class="avoindata-explore-card mt-2">
+        <div class="avoindata-explore-card-icon avoindata-explore-apiset">
           <img src="/themes/avoindata/images/exchange.svg"/>
         </div>
 
-        <div class="col-md-8 avoindata-explore-card-content pl-3">
-          <div class="row container-fluid">
+        <div class="avoindata-explore-card-content">
+          <div class="avoindata-explore-card-title">
             <h3>
               {% trans %}
                 APIs
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
+
+          <div class="avoindata-explore-card-text">
             <i class="far fa-chevron-right"></i> 
-            <a href="/data/{{ language }}/apiset">
+            <a href="/data/{{ language }}/dataset">
```

Comment:
Is this link updated from apiset => dataset for a purpose?

Reply:
Fixed in [a6d7b26](https://github.com/vrk-kpa/opendata/pull/1730/commits/a6d7b268a8685aa016145c4124821dccf0eecd1f)